src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -999,7 +1004,15 @@
      "CREATE INDEX idx_resource_events_resource_title ON resource_events(resource_title)"
      "CREATE INDEX idx_resource_events_status ON resource_events(status)"
      "CREATE INDEX idx_resource_events_timestamp ON resource_events(timestamp)"
-     "ALTER TABLE resource_events ADD CONSTRAINT resource_events_report_id_fkey FOREIGN KEY (report_id) REFERENCES reports(id) ON DELETE CASCADE"))
+     "ALTER TABLE resource_events ADD CONSTRAINT resource_events_report_id_fkey FOREIGN KEY (report_id) REFERENCES reports(id) ON DELETE CASCADE")
+
+(sql/do-commands
+      ;; recreate indices
+      "ALTER TABLE certnames ADD CONSTRAINT certnames_pkey PRIMARY KEY (name)"
```

Comment:
We've missing the index for report_id ... https://gist.github.com/kbarber/2cfb762adcf87a1b243c